locales-src/newtab.ftl

```diff
@@ -83,6 +83,13 @@ newtab-menu-delete-history = Delete from History
 newtab-menu-save-to-pocket = Save to { -pocket-brand-name }
 newtab-menu-delete-pocket = Delete from { -pocket-brand-name }
 newtab-menu-archive-pocket = Archive in { -pocket-brand-name }
+newtab-menu-show-privacy-info = Our sponsors & your privacy
+
+## newtab privacy info modal
+newtab-privacy-modal-done = Done
+newtab-privacy-modal-header = Your privacy matters.
+newtab-privacy-modal-paragraph = In addition to dishing up captivating stories, we also show you relevant, highly-vetted content from select sponsors. Rest assured, your browsing data never leaves your personal copy of Firefox—we don’t see it, and our sponsors don’t either.
```

Comment:
You can use multiline strings (indent with spaces) when the text gets this long.

```
 newtab-privacy-modal-paragraph =     
    In addition to dishing up captivating stories, we also show you relevant,
    highly-vetted content from select sponsors. Rest assured, your browsing
    data never leaves your personal copy of Firefox—we don’t see it, and our
    sponsors don’t either.
```

Never hard-code Firefox in strings, there are brand terms that can be used instead (`{ -brand-product-name }`).

The lack of spaces around the hyphen doesn't look correct?
https://design.firefox.com/photon/copy/punctuation.html

Reply:
GTK. Will use a spaced em dash per style guide.